<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Credit</class>
 <widget class="QWidget" name="Credit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: white;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true"></string>
         </property>
         <property name="text">
          <string>Сумма кредита</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_1">
         <property name="styleSheet">
          <string notr="true">QLineEdit { 
border-radius: 12px;
border: 1px solid rgb(222, 222, 222);

padding-left: 12px;
height: 40px;
}

QLineEdit:focus {
border: 1px solid #bfff6a;
}</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="7" column="0">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true"></string>
           </property>
           <property name="text">
            <string>Срок кредита (мес)</string>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="label_10">
           <property name="styleSheet">
            <string notr="true"></string>
           </property>
           <property name="text">
            <string>Тип платежей</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_4">
           <property name="styleSheet">
            <string notr="true"></string>
           </property>
           <property name="text">
            <string>Ежемесячный платеж</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Minimum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="2">
          <widget class="QLineEdit" name="lineEdit_3">
           <property name="styleSheet">
            <string notr="true">QLineEdit { 
border-radius: 12px;
border: 1px solid rgb(222, 222, 222);

padding-left: 12px;
height: 40px;
}

QLineEdit:focus {
border: 1px solid #bfff6a;
}</string>
           </property>
          </widget>
         </item>
         <item row="10" column="2">
          <widget class="QComboBox" name="comboBox">
           <property name="styleSheet">
            <string notr="true">
height: 40px;</string>
           </property>
           <item>
            <property name="text">
             <string>Аннуитетные</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Дифференцированные</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QLabel" name="res_1">
           <property name="styleSheet">
            <string notr="true">font-weight: bold;
</string>
           </property>
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="11" column="0">
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_9">
           <property name="styleSheet">
            <string notr="true"></string>
           </property>
           <property name="text">
            <string>Долг + проценты</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true"></string>
           </property>
           <property name="text">
            <string>Процентная ставка</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="6" column="2">
          <widget class="QLabel" name="res_2">
           <property name="styleSheet">
            <string notr="true">font-weight: bold;
</string>
           </property>
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="8" column="2">
          <widget class="QLabel" name="res_3">
           <property name="styleSheet">
            <string notr="true">font-weight: bold;
</string>
           </property>
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <widget class="QLabel" name="res_4">
           <property name="styleSheet">
            <string notr="true">font-weight: bold;
</string>
           </property>
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLineEdit" name="lineEdit_2">
           <property name="styleSheet">
            <string notr="true">QLineEdit { 
border-radius: 12px;
border: 1px solid rgb(222, 222, 222);

padding-left: 12px;
height: 40px;
}

QLineEdit:focus {
border: 1px solid #bfff6a;
}</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_6">
           <property name="styleSheet">
            <string notr="true"></string>
           </property>
           <property name="text">
            <string>Проценты</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <spacer name="verticalSpacer_6">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="styleSheet">
        <string notr="true">QPushButton {
border-radius: 12px;
background: #f4f4f4;
font-size: 16px;

font-weight: bold;
height: 60px;
}
QPushButton:pressed {
background: rgb(219, 220, 224);
}

QPushButton:hover {
background:rgb(240, 240, 240);
}</string>
       </property>
       <property name="text">
        <string>Рассчитать</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
